README.md

```diff
@@ -1,2 +1,314 @@
+
+4) Commit the changes
+5) Revisit the onboarding PR and notice how the onboarding PR automatically updates to reflect the changes you made to the configuration 
+
+<img width="830" alt="onboarding warning hourly update" src="https://user-images.githubusercontent.com/102745725/178960884-40077a5c-8fe1-422f-81c1-567ea1e6619b.png">
+
+> **Note**
+> May take a few moments to update.
+
+6) Merge the onboarding pull request.
+
+**Congratulations! You have successfully onboarded Renovate.** 🎈
+
+## Part 3 - Getting to know Renovate’s update PRs
+
+Now that you have merged the onboarding PR, Renovate will generate Update PRs to the most recent dependency version based on your configuration. 
+
+> **Note**
+> PRs may take a couple of minutes to appear
+
+Here we will review the basic concepts of Renovate update PRs and merge it.
+
+- By default, Renovate will create up to 2 update PRs per hour. However, if you completed the onboarding section of this tutorial, Renovate will now create 3 PRs.
+- You should already see notifications for these pull requests in the `Pull Requests` section of the repo.
+
+Let’s go ahead and take a look at a Renovate update PR:
+1) Navigate to the `Pull requests` section and open - `Update dependency lodash to x.y.z`
+
+<img width="1256" alt="open PRs" src="https://user-images.githubusercontent.com/102745725/178967929-690b3866-190b-4576-a961-981ce78cbd1b.png">
+
+### Each update PR contains:
+- Dependency information (name and version changes)
+- [Merge Confidence](https://docs.renovatebot.com/merge-confidence/) values
+- Up-to-date release notes
+- Renovate configuration-related info
+- Option to rebase PR
+- Link to view Renovate logs
+
+![image](https://user-images.githubusercontent.com/42116482/178495994-7cce93f1-db65-4f09-b682-7506dc242fdc.png)
+
+- Renovate’s update PRs will update the relevant dependency across your entire repo. In our RenovateTutorial repo, this will be both the `package.json` file and the `package-lock.json` file:
+
+<img width="1488" alt="file diff lodash" src="https://user-images.githubusercontent.com/102745725/178968020-865560f7-173c-4e9e-a073-488147dbb137.png">
+
+2) Merge this pull request
+
+>Note - Renovate is highly configurable and supports:
+>
+>- On-demand PR creation.
+>- Automatic merging of PRs.
+>- Settings for specific dependencies/package managers.
+>- Scheduling.
+>- Grouping. 
+>
+>  All the above and more will be discussed in future parts of the tutorial.
+
+**Congratulations! You have now updated your first dependency with Renovate.** 🎈
+
+## Part 4 - Dependency Dashboard
+
+Renovate’s Dependency Dashboard is a GitHub Issue that enables you to manage and monitor Renovate’s activity in your repo. In this section, we will go over some of its main functionalities and capabilities.
+
+Let’s begin by creating and enabling the Dependency Dashboard. Since GitHub defaults to disable `issues` on forked repositories, we need to enable it on the forked RenovateTutorial repo:
```

Comment:
```suggestion
Let’s begin by creating and enabling the Dependency Dashboard.
Since GitHub defaults to disable `issues` on forked repositories, we need to enable it on the forked RenovateTutorial repo:
```